src/azure-cli-core/azure/cli/core/__init__.py

```diff
@@ -851,3 +853,13 @@ def get_default_cli():
                  logging_cls=AzCliLogging,
                  output_cls=AzOutputProducer,
                  help_cls=AzCliHelp)
+
+
+def _configure_knack():
+    """Override consts defined in knack to make them Azure CLI-specific."""
+    from knack.util import status_tag_messages
+    ref_message = "Reference and support levels: https://aka.ms/CLI_refstatus"
+    # Override the preview message
+    status_tag_messages['preview'] = "{} is in preview and under development. " + ref_message
```

Comment:
```suggestion
    status_tag_messages['preview'] = "{} is in preview. " + ref_message
```
I noticed that in knack there's no 'under development' for preview message.

Reply:
It is not necessary for Azure CLI and Knack to have the same message. The message I used in this PR has been approved by PMs and UX team.